packages/extension/src/view/popup/app.tsx

```diff
@@ -65,7 +65,7 @@ const App: React.FC = () => {
 
   if (onChromeUrl) {
     return (
-      <div className="w-full h-full flex justify-center items-center flex-col z-1 relative">
+      <div className="w-full h-full flex justify-center items-center flex-col z-1">
```

Comment:
It was updated in https://github.com/GoogleChromeLabs/ps-analysis-tool/commit/789a40aa38ca781b9830b38a0a24f94fc59e657e

Reply:
Resolved